src/server/pegasus_server_write.h

```diff
@@ -84,6 +94,10 @@ class pegasus_server_write : public dsn::replication::replica_base
     int64_t _decree;
 
     const bool _verbose_log;
+
+    typedef std::map<dsn::task_code, std::function<int(pegasus_server_write *, dsn::message_ex *)>>
+        single_put_rpc_map;
+    static single_put_rpc_map _single_put_methods;
```

Comment:
Called them `non_batch_writes_map`, and `_non_batch_write_handlers`. The other methods are not "put". It's somewhat ambiguous.